CHANGES.rst

```diff
@@ -17,6 +17,8 @@ In development
   order of task completion rather than the order of submission.
   https://github.com/joblib/joblib/pull/1463
 
+- Vendor cloudpickle 3.0.0 and drop support for Python 3.7.
```

Comment:
maybe add a ref to the PR so it is consistent with the other and easy to reach?

Reply:
Done.